tests/demo_extensions_additional/copier.yml

```diff
@@ -0,0 +1,3 @@
+_extensions:
```

Comment:
These days I'm more in favor of building the code tree in the test module. It seems cleaner and easier to reference the test with the template. I've done that for you.

Reply:
> I've done that for you.

Not sure to understand! I don't any new commit from you on this branch?

Reply:
Ah! I forgot to upload it.